feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewUiState.kt

```diff
@@ -27,7 +27,8 @@ data class PreviewUiState(
     val currentCameraSettings: CameraAppSettings,
     val lensFacing: Int = CameraSelector.LENS_FACING_BACK,
     val videoRecordingState: VideoRecordingState = VideoRecordingState.INACTIVE,
-    val quickSettingsIsOpen: Boolean = false
+    val quickSettingsIsOpen: Boolean = false,
+    val captureIsSuccessful: Boolean = false
```

Comment:
TBH, I added this just so that it's a bit more easier during test (ensuring my changes aren't actually breaking image capture itself) and thought it might be better for everyone to keep it xD

But I didn't really want to do too many changes regarding this since it doesn't really relate to this PR, just did the bare minimum for testing purpose. For now, I have refactored this further to add the error case too, but later on, I guess we need to handle different types of error cases too and do a proper design for this.

Reply:
Thanks. That sounds great. I just didn't want to set a precedent that error cases shouldn't be handled. Your changes look good.